cmd/juju/controller/addmodel_test.go

```diff
@@ -42,11 +43,15 @@ var _ = gc.Suite(&AddModelSuite{})
 
 func (s *AddModelSuite) SetUpTest(c *gc.C) {
 	s.FakeJujuXDGDataHomeSuite.SetUpTest(c)
+
+	agentVersion, err := version.Parse("2.2-rc1")
```

Comment:
If we are going to set a fake agent version, I'd rather it not match version.Current.

Perhaps choose something like 2.55 ?

Otherwise it is too easy to miss the cases where the current version is being used insetad of the fake one.